### PR TITLE
docs: add NixOS installation instructions to README

### PR DESCRIPTION
**Pull request by ob7**
Add NixOS package installation section under a new "Package Managers" category in the installation instructions. This reflects tenere's recent addition to nixpkgs (NixOS/nixpkgs/pull/363556)

Resolves pythops/tenere#43

### CODE REVIEW

**Comment by ob7** [comment]
hmm .. just realized crates is considered a "package manager" so perhaps I need a different title than one I used here ...

**Comment by ob7** [comment]
or maybe we don't even need a section for system package managers if nixos is only one and other distros probably just use crates?

**Comment by pythops** [comment]
Can it be installed like this ?
```
nix-env -iA nixpkgs.tenere
```

**Comment by ob7** [comment]
Any of the Nix install methods should work, however its only available in master until I backport it to a stable release like 24.11, 24.05, etc. 

So thinking about it, even though tenere is in nixpkgs now, maybe hold off merging this until it's in one of the main release branches so that most ppl using nixos/nixpkgs will actually see it show up for them.  As of now one needs to clone the nixpkgs repo and install it using the master branch.  It's not yet visible in any of the channels.

**Comment by pythops** [comment]
You are the maintainer of the nixpackage :smile: You do what you think is right buddy :)

**Comment by ob7** [comment]
I wanted to make sure you weren't aware of any issues running tenere using deps generated for cargo.lock v3.  I'll assume not, I just like to be thorough

**Comment by pythops** [comment]
not that I know no.

**Comment by ob7** [comment]
@pythops - tenere has been successfully backported to nixos release-24.11 (https://github.com/NixOS/nixpkgs/pull/364564).  It can now be installed via normal release channels, as well as via ```nix-env -iA nixpkgs.tenere``` on non-nixos systems running nixpkgs!  I tested on debian.  

I still have a PR opened for backport to release-24.05 (https://github.com/NixOS/nixpkgs/pull/364671) for ppl running older version.

File: README.md
Comment (pythops) [inline README.md:56]: maybe you can have it in multiple lines, Something like

```
Or as following for Non-NixOs systems
\```
nix-env -iA nixpkgs.tenere
\```
```

**Comment by ob7** [comment]
Hows this?:
https://github.com/pythops/tenere/blob/927571cd79fb7cfbc9c0adab08eaa606499d8913/README.md?plain=1#L56-L59

**Review by pythops — APPROVED** [review]

**Comment by pythops** [comment]
Thanks buddy for the PR and the work done to add support for nix :pray:

**Pull request merged**
